solutions/tests/test_add_numbers.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Test module for add_numbers function.
+
+Test categories:
+    - Standard cases: typical numbers
+    - Edge cases: zero, negative numbers
+    - Defensive tests: wrong input types, assertions
+
+Created on 2024-12-06
+Author: GitHub Copilot
+"""
+
+import unittest
+from solutions.add_numbers import add_numbers
```

Comment:
```suggestion
from ..add_numbers import add_numbers
```

Why don't you try this neet relative importing style.